assets/styles/style.css

```diff
@@ -41,32 +41,13 @@ ul {
   padding: 10px 40px;
 }
 
-a {
-  /* color: #e12a20; red, feels like too much */
-  color: var(--main-txt-color, #0476A3);
-  text-decoration: none;
-}
-
-a:hover, a:focus {
-  color: var(--main-txt-color-dark, #026494);
-  text-decoration-line: underline;
-  text-decoration-style: dashed;
-  text-decoration-skip-ink: none;
-}
-
-a.button, .btn-primary {
-  box-shadow: 0 0 5px var(--box-shadow-color, rgba(19, 140, 171, 0.8));
-  border-radius: 5px;
-  text-decoration: none;
-  font-family: var(--secondary-font);
-  padding: 20px;
-  color: var(--main-txt-color, #0476A3);
-}
-
-a.button:hover, a.button:focus, .btn-primary:hover, .btn-primary:focus {
-  background-color: var(--secondary-bg-color, #0476A3);
-  color: var(--secondary-txt-color, #ffffff);
-  text-decoration: none;
+img {
+  display: block;
+  max-width: 750px;
+  margin-right: auto;
+  margin-left: auto;
+  margin-top: 50px;
+  margin-bottom: 50px;
```

Comment:
There is a placeholder image that comes over with Paypal, and it's causing the Donate button in the footer to be wonky.

I found if I removed the margin-top and margin-bottom lines of css here, that it looked okay, and I _think_ the rest of the images also worked okay.

Could we take a look at this, or some other solution you prefer?

Here's a screen shot of the css bug.
<img width="1140" alt="Screen Shot 2020-03-01 at 4 06 14 PM" src="https://user-images.githubusercontent.com/11672661/75634992-24ff5c00-5bd8-11ea-8601-24df25370d27.png">

Reply:
Oooohh good eye, I had totally missed this!

Reply:
Ooohh good call, I wasn't seeing this in Firefox!

Reply:
I wound up figuring out a weird fix for this because removing the margins made it look good on Chrome but messed it up on Firefox. 🙃 There was a 1-pixel image that I wound up doing `display: none` on, and that _seems_ to have fixed it. The donate link still works for me, so let me know if you realize that this actually wasn't a good fix!